V2/tezex/src/App.tsx

```diff
@@ -1,21 +1,31 @@
 import React, { FC } from "react";
-import './App.css';
+import "./App.css";
+
 
 import { Layout } from "./components/ui/views/Layout";
+import { Outlet } from "react-router-dom";
 
+/*
+<Router basename={process.env.PUBLIC_URL}>
+	<header className="App-header">
+			<Layout>
+				<Routes>
+					<Route path="/Home" element={<Home />} />
+					<Route path="/Analytics" element={<Analytics />} />
+					<Route path="/About" element={<About />} />
+				</Routes>
+			</Layout>
+	</header>
+</Router>
+*/
```

Comment:
maybe we should clean up these comments if they won't be used

Reply:
acknowledged